tools/coreml/converter/_layers.py

```diff
@@ -314,7 +314,12 @@ def convert_convolution(net, node, module, builder):
         Wb = None
 
     channels = W.shape[1]
-    stride_height, stride_width = literal_eval(param['stride'])
+
+    stride_height = 1
+    stride_width = 1
+    if 'stride' in param.keys():
+        stride_height, stride_width = literal_eval(param['stride'])
+
     kernel_height, kernel_width = literal_eval(param['kernel'])
```

Comment:
Shall we do the same initialization for kernel height and kernel width as stride height and stride weight?

Reply:
Thanks for reviewing Jiajie.

Kernel is a required parameter for the de/convolutional layer so we can be certain that it'll exist whereas the same thing can't be said about pad/stride. See more **[here](https://mxnet.incubator.apache.org/api/python/symbol.html#mxnet.symbol.Convolution).**